src/services/llms/anthropic.py

```diff
@@ -31,8 +31,7 @@ def create_prompt(self, file: PatchedFile, hunk: Hunk, pr_details: PRDetails) ->
           * Security vulnerabilities
           * Performance issues
           * Code structure and maintainability
-        
-        Please provide your review in {Config.HUMAN_LANGUAGE} language.
+        - IMPORTANT: Provide your review in {Config.HUMAN_LANGUAGE} language
```

Comment:
### Potential Issue

The updated `system` prompt is very vague and may not provide enough context for the AI model. Consider including more details that can guide the model effectively in its responses. This change might lead to suboptimal output quality.